### PR TITLE
Improve Address Error Messages

### LINKED ISSUE #343: Update multi-addresses error message
When a user tries to create an Address record from the Addresses related list in Contact or Account, and multi-addresses is not enabled for Contacts or for that Account record type, respectively, an error is displayed in the UI. The text of this error should be updated to: [Docs to provide text]

### PR DESCRIPTION
**Pull request by davidjray**
# Critical Changes

# Changes
Separate error messages for Account and Contact when trying to create a child address without Multi-Addresses enabled.

# Issues Closed
Fixes #343.

### CODE REVIEW

**Comment by davidjray** [comment]
This pull request also includes the CumulusCI 2 yml file because I needed it for this work as well. That makes #346 redundant.

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -134,7 +134,15 @@
...
         <protected>true</protected>
         <shortDescription>addrNotEnabled</shortDescription>
-        <value>Address management not enabled for this object type in settings.</value>
+        <value>The ability to add multiple addresses is disabled for this Account record type. To enable it, go to HEDA Settings, click on Accounts and Contacts, and select the appropriate checkbox next to Account Types with Multi-Addresses Enabled.</value>
+    </labels>
+    <labels>
+        <fullName>addrNotEnabledContact</fullName>
+        <categories>Address Management</categories>
+        <language>en_US</language>
+        <protected>true</protected>
```
Comment (paulmalenke) [inline src/labels/CustomLabels.labels]: @ceiroa I cannot remember if we said we would move forward with true or false here. Did we document that somewhere?

File: src/package.xml [hunk 2]
```diff
@@ -318,6 +318,7 @@
         <members>RelationshipsLookupDescription</members>
         <members>addrHhAdmAccountOnly</members>
         <members>addrNotEnabled</members>
+        <members>addrNotEnabledContact</members>
```
Comment (paulmalenke) [inline src/package.xml]: I believe this is the only line that should be committed for package.xml. The extra lines are not needed.

File: src/classes/ADDR_Addresses_TDTM.cls [hunk 3]
```diff
@@ -1,10 +1,10 @@
 /*
     Copyright (c) 2014, Salesforce.org
     All rights reserved.
-
```
Comment (paulmalenke) [inline src/classes/ADDR_Addresses_TDTM.cls]: Loving all the whitespace deletion!

File: src/labels/CustomLabels.labels (hunk 1 above)
Comment (ceiroa) [inline src/labels/CustomLabels.labels]: They should be protected unless there's a good reason for them not to be (if we wanted users to be able to change them). The reason why it's better for us to make them protected is that protected labels can be deleted from the package if we don't need them any more.

**Comment by ceiroa** [comment]
I might be missing something, but I don't see where the new error message is being used...

**Comment by paulmalenke** [comment]
@ceiroa You might have to "Load diff" for src/classes/ADDR_Addresses_TDTM.cls. The diff is large so it does not show it unless you click the link in the Files changed.

**Comment by ceiroa** [comment]
@paulmalenke You are right, it wasn't loading it. But now that I see it I still think it's only using one of the error labels...

**Pull request merged**